src/components/navigation/Link.tsx

```diff
@@ -23,35 +28,45 @@ const Link = function Link({
   elementRef,
 
   underline = 'none',
-  color = 'brand',
+  color,
+  unstyled = false,
+  variant = 'brand',
 
   ...htmlAttributes
 }: LinkProps) {
+  // Map from deprecated `color` prop to `variant` if `color` is present
+  const theme = typeof color === 'string' ? color : variant;
+  const styled = !unstyled;
+  const themed = styled && theme !== 'custom';
+
   return (
-    <LinkBase
+    <a
       data-component="Link"
+      rel="noopener noreferrer"
       {...htmlAttributes}
-      classes={classnames(
-        'rounded-sm',
-        // NB: Base classes are applied by LinkBase
+      className={classnames(
         {
-          // color
-          'text-brand hover:text-brand-dark': color === 'brand', // default
-          'text-color-text-light hover:text-brand': color === 'text-light',
-          'text-color-text hover:text-brand-dark': color === 'text',
+          'focus-visible-ring rounded-sm': styled,
+          // underline
+          // TODO: Underline should be controlled by `variant` and should default
+          // to `always`
```

Comment:
I want to acknowledge that `underline` is a problematic prop that doesn't fit into this model neatly. Which means I think it should go in the short future.

Reply:
My intuition is that a single enum "variant" is probably not going to be enough to accommodate the different "standard"/built-in styles that a component may have. I wouldn't be surprised if we need flag props for example to turn certain elements of the standard style on/off. If the component is unstyled or using a custom variant or size, then these props wouldn't apply though.